Build/GameSettings.cc

```diff
@@ -186,7 +186,7 @@ void InitGameOptions()
 	gGameOptions.fSciFi						 = TRUE;
 	gGameOptions.ubDifficultyLevel = DIF_LEVEL_EASY;
 	//gGameOptions.fTurnTimeLimit		 = FALSE;
-	gGameOptions.fIronManMode			 = FALSE;
+	gGameOptions.ubGameSaveMode			 = DIF_CAN_SAVE;
```

Comment:
above you're checking it against GIO_* constants. Are both sets needed, is there a bug somewhere?